devices/At24CXX/AT24Base.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.At24CXX
+{
+    /// <summary>
+    /// Base class for common functionality of the AT24C serices of devices.
+    /// </summary>
+    public abstract class AT24Base : IDisposable
+    {
+        /// <summary>
+        /// Default I2C address for AT24Cx familly
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x50; // 01010000
+
+        private bool _isDisposed;
+        private readonly I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Number of pages available on the device.
```

Comment:
All properties comments should start with 'Gets' if it's only a public get like here or 'Gets or sets' if both.